wled00/data/index.htm

```diff
@@ -2121,6 +2123,22 @@
 	return mergeDeep(target, ...sources);
 }
 
+function hcUpdate() {
+	const regex = /^#([0-9a-f]{6})$/i;
+	if (regex.test(document.getElementsByName("HC")[0].value)) {
+		document.getElementById("invalidHexColor").style.display = "none";
+		cpick.color.set(document.getElementsByName("HC")[0].value);
```

Comment:
Since I have been working on JavaScript minimization, here are some tips to make this code better. Not sure it will improve the chance this is merged. 

Use the `d` variable instead if `document` to keep JavaScript size down. Also avoid accessing the DOM multiple of times. Save the value of `document.getElementsByName("HC")` and reuse. ie

```javascript
function hcUpdate() {
	const regex = /^#([0-9a-f]{6})$/i;
	const hexcolor = document.getElementsByName("HC");
	if (regex.test(hexcolor[0].value)) {
		...
		cpick.color.set(hexcolor[0].value);

```

Reply:
Thanks @pbolduc, but I'm not sure if this PR is necessary anymore. I read in the [0.10.2 release notes](https://github.com/Aircoookie/WLED/releases/tag/v0.10.2) that it adds "Option for HEX color entry and RGB sliders." I don't currently have a WLED instance up and running, is anyone able to confirm that this is the same feature that's in this PR? If so, I can go ahead and close this.

Reply:
Seems you are correct, I found this code but I have not validated on a running WLED instance,

https://github.com/Aircoookie/WLED/blob/503b8cbf349158ecfaf1fa7969e745809a53d142/wled00/data/index.htm#L943

Reply:
Here is from the latest source:
![image](https://user-images.githubusercontent.com/1844480/91896678-1a3c8700-ec4e-11ea-8d64-d41ce0c4c9fb.png)
clicking on the icon,
![image](https://user-images.githubusercontent.com/1844480/91896716-29bbd000-ec4e-11ea-969f-fe89e3073ae0.png)